concordia/static/js/action-app/index.js

```diff
@@ -218,8 +218,6 @@ export class ActionApp {
             button.classList.toggle('active', button.value == newMode);
         });
 
-        $$('.current-mode').forEach(i => (i.textContent = this.currentMode));
```

Comment:
We still have this element in the DOM — should we remove it from `action-app.html` or do we need a separate place to update it?

Reply:
I removed it from the DOM.